New backoffice: Fix a few glitches in the dashboard tutorial (#5446)
umbraco/UmbracoDocs: new-backoffice/tutorials/creating-a-custom-dashboard.md

This guide takes you through the steps to set up a Custom Dashboard in Umbraco.

> The guide is divided into several parts. In this first part will go through:

Reviewer: ```suggestion
The guide is divided into four parts. The first part will go through the following:
```
